subjects/devops/find-files/README.md

```diff
@@ -0,0 +1,49 @@
+## find-files
+
+### Instructions
+
+"start finding ..."
+
+Create a file `find-files.sh`, which will look for and show, in the current directory and its sub-folders:
```

Comment:
Ok. In any case, I feel the ask should be: 

everything that starts with an `a` **and**,
all the files ending with a `z`

Showing just one of the two would not be enough to pass this exercise

Reply:
i delete the `or` in the second line but i kept the `or` in the first line you can check the script
you can check the solution file its check with or not and

Reply:
For clarity, I would rephrase it as follow

```
everything that starts with an `a` or files ending with a `z`
```

In any case, the example below clarify what is supposed to have as a result of the script

Reply:
done, i explained cause the students will join the branch with no experience in linux so this will be hard for them to know what the different between all files and everything in linux
thank you